sermant-integration-tests/xds-service-test/xds-service-integration-test/src/test/java/io/sermant/xds/service/discovery/XdsServiceDiscoveryTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (C) 2024-2024 Sermant Authors. All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package io.sermant.xds.service.discovery;
+
+import io.sermant.xds.service.utils.HttpRequestUtils;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfSystemProperty;
+
+/**
+ * xDS service discovery test
+ *
+ * @author daizhenyu
+ * @since 2024-07-15
+ **/
+public class XdsServiceDiscoveryTest {
+    /**
+     * one server instance
+     */
+    @Test
+    @EnabledIfSystemProperty(named = "xds.service.integration.test.type", matches = "DISCOVERY_ONE_SERVER_INSTANCE")
+    public void testDiscoveryWithOneServerInstance() {
+        String[] resultStrings = splitResult(HttpRequestUtils.doGet("http://127.0.0.1:8080/hello"));
+        if (resultStrings.length != 2) {
+            Assertions.assertEquals(0, 1, "The returned result format is incorrect.");
```

Comment:
Assertions.assertEquals(2, resultStrings.length, "The returned result format is incorrect.");

Reply:
done